Cargo.toml

```diff
@@ -108,6 +119,21 @@ tcp = [
     "tokio/time",
 ]
 
+# Tracing layers
+layers = [
+    "log",
+    "opentelemetry",
+    "opentelemetry-jaeger",
+    "serde_json",
+    "reqwest",
+    "tracing",
+    "tracing-core",
+    "tracing-log",
+    "tracing-opentelemetry",
+    "tracing-subscriber",
+    "tracing-tree",
```

Comment:
i don't think that the feature that enables `tracing` instrumentation in Hyper should enable all of these dependencies. suppose I want to log `tracing` spans from hyper using `tracing-subscriber`, in the plaintext format. in that case, i would want to enable `tracing` instrumentation in `hyper`...but i would not want `hyper` to suddenly depend on `log`, `serde_json`, `reqwest`, `tracing_opentelemetry`, `opentelemetry`, `opentelemetry-jaeger`, and `tracing-tree`.

Reply:
> i don't think that the feature that enables `tracing` instrumentation in Hyper should enable all of these dependencies. suppose I want to log `tracing` spans from hyper using `tracing-subscriber`, in the plaintext format. in that case, i would want to enable `tracing` instrumentation in `hyper`...but i would not want `hyper` to suddenly depend on `log`, `serde_json`, `reqwest`, `tracing_opentelemetry`, `opentelemetry`, `opentelemetry-jaeger`, and `tracing-tree`.

Summarizing - response to the question of introducing a feature: No objection, in fact please consider features that allow more fine grained control. Accurate summary?